barretenberg/cpp/src/barretenberg/relations/ecc_op_queue_relation.hpp

```diff
@@ -38,6 +37,8 @@ template <typename FF_> class EccOpQueueRelationImpl {
      * The first four sub-relations check that the values in the conventional wires are identical to the values in the
      * ecc op wires over the portion of the execution trace representing ECC op queue gates. The next four check
      * that the op wire polynomials are identically zero everywhere else.
+     * @note This relation utilizes the shifted wires so that the ecc op wires can store the data begining at index 0,
+     * unlike the wires which contain an initial zero row to facilitate the left-shift-by-1 needed by other relations.
```

Comment:
isn't it right-shift-by-1?

Reply:
I guess it depends how your reference but the things we label `*_shift` are the left shift by 1 of `*`

Reply:
so `*` is the right shift by 1 of `*_shift` if you like...